data/event-schemas/schema/m.key.verification.start$m.sas.v1.yaml

```
@@ -42,7 +42,10 @@ properties:
         type: array
         description: |-
           The message authentication codes that the sending device understands.
```

We're not negotiating codes per se, but rather methods for generating them.

```suggestion
          The message authentication code methods that the sending device understands.
```

similarly in `m.key.verification.accept`.